### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by dungnthfx00445**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by dungnthfx00445 — COMMENTED** [review]
Good!

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (dungnthfx00445) [inline index.html:50]:
``` 
It should be bolded
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

Comment (tienSyBD) [inline index.html:50]: Replace
```
<h2 hidden>Game over, refresh to play again!</h2>
```

Comment (tienSyBD) [inline index.html:50]:
```
<h2 hidden>Game over, refresh to play again!</h2>
```

Comment (tienSyBD) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again!</h2>
```

**Pull request merged**